includes/class.llms.frontend.forms.php

```diff
@@ -73,11 +73,10 @@ public function take_quiz() {
 	 * Marks lesson as complete and returns completion message to user
 	 *
 	 * @return void
+	 * @version 3.2.4
 	 */
 	public function mark_complete() {
```

Comment:
this function can be removed completely

the `mark_complete` function in the frontend forms class should hold the logic here